math/vecops/inc/ROOT/RVec.hxx

```diff
@@ -1263,6 +1263,10 @@ Its storage is contiguous in memory and its interface is designed such to resemb
 of the stl vector. In addition the interface features methods and external functions to ease
 the manipulation and analysis of the data in the RVec.
 
+\note ROOT::VecOps::RVec can also be spelled simply ROOT::RVec, and shorthand aliases such as ROOT::RVecI, ROOT::RVecD
```

Comment:
Since this is the main place in the docs where you mention the availability of such aliases, I would list them all. I'm not sure the `using` declarations defined below will all get picked up by doxygen and still they would be listed as separate boxes whereas if you list them all here it would be more compact to read

Reply:
good point, doxygen does not do a good job of advertising type aliases

Reply:
done